mailpoet/assets/js/src/wizard/steps/pitch_mss_step.tsx

```diff
@@ -0,0 +1,35 @@
+import { useRouteMatch, Switch, Route } from 'react-router-dom';
+import { MSSStepFirstPart } from './pitch_mss_step/first_part';
+import { MSSStepSecondPart } from './pitch_mss_step/second_part';
+import { MSSStepThirdPart } from './pitch_mss_step/third_part';
+
+type WelcomeWizardPitchMSSStepPropType = {
+  subscribersCount: number;
+  finishWizard: (redirect_url?: string) => void;
+};
+
+function WelcomeWizardPitchMSSStep({
+  subscribersCount,
+  finishWizard,
+}: WelcomeWizardPitchMSSStepPropType): JSX.Element {
+  const { path } = useRouteMatch();
+
+  return (
+    <Switch>
+      <Route path={`${path}/part/2`}>
+        <MSSStepSecondPart finishWizard={finishWizard} />
+      </Route>
+      <Route path={`${path}/part/3`}>
+        <MSSStepThirdPart finishWizard={finishWizard} />
+      </Route>
+      <Route path={[`${path}`, `${path}/part/1`]}>
```

Comment:
Another thing that I don't like about this is the fact that I had to use two paths here and because of the root path, I had to place the route for the first past last.

Reply:
Well if we want both of these routes to render them same component, then this is the way to go, but the other way around would be to redirect the route path to `${path}/par/1`, something like 
```
<Route exact path={`${path}`}>
  <Redirect to={`${path}/part/1`} /> 
</Route>

```

Reply:
Thanks for your input. I think it is much cleaner than what I had and the `<Redirect>` component is what I was missing to be able to implement it.

I had to change `<Redirect to={`${path}/part/1`} />` to `<Redirect to={`/steps/${step}/part/1`} />` and call `useParams()` to get the value of `step`. I believe this is because `<Redirect>` expects a URL and not the route match string.

Since this is the review of a draft PR, to avoid polluting the commit history and since this PR will need to be reviewed again, I'm opting to amend the commits that already existed with the changes instead of creating new commits.